float64 car_id
float64 x_pos
float64 y_pos
